nvdaHelper/remote/textFromIAccessible.cpp

```diff
@@ -13,13 +13,53 @@ This license can be found at:
 
 #include "textFromIAccessible.h"
 #include <string>
+#include <vector>
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
 #include <atlcomcli.h>
 #include <ia2.h>
 #include <common/ia2utils.h>
 
 using namespace std;
+auto constexpr OBJ_REPLACEMENT_CHAR = L'\xfffc';
+
+
+bool isEmpty(CComBSTR& val) {
+	if (!val) {
+		return true;
+	}
+	for (int i = 0; val[i] != L'\0'; ++i) {
+		if (val[i] != OBJ_REPLACEMENT_CHAR && !iswspace(val[i])) {
+			return false;
+		}
+	}
+	return true;
+}
+
+bool appendNameDescription(CComPtr<IAccessible> pacc, wstring& textBuf) {
+	bool gotText = false;
+	CComVariant varChild;
+	varChild.vt = VT_I4;
+	varChild.lVal = 0;
+
+	CComBSTR val;
+	pacc->get_accName(varChild, &val);
+	bool valEmpty = isEmpty(val);
+	if (!valEmpty) {
+		gotText = true;
+		textBuf.append(val);
+		textBuf.append(L" ");

```

Comment:
This appends a space even if there is no description later. Perhaps we could remove this line, and lower down once we have a valid description, just before setting gotText to true, append the space if gotText was already true. 
If you just copied the original logic however, I'm happy for you to leave this.

Reply:
I believe this matches the logic from before (see https://github.com/nvaccess/nvda/pull/13127/files#diff-a8111ab994be8314dfa43a5f725fd2969ac65ad8f68f899f4f7dabf6088b9bb1L137) I'll double check this.

Reply:
I can't see any differences in the logic.

Reply:
Yep, that is fine. In that case just leave it.